frontend/app/src/store/modules/base.js

```diff
@@ -93,13 +97,14 @@ const actions = {
 }
 
 const mutations = {
-  [types.LOGIN] (state, { token, username, isSuperuser, channel }) {
+  [types.LOGIN] (state, { token, username, isSuperuser, channel, permissions }) {
     state.token = token
     state.user = username
     state.isSuperuser = isSuperuser
     state.channel = channel
     state.loginSuccess = true
     state.loginMessage = ''
+    Vue.set(state, 'permissions', permissions)
```

Comment:
Why Vue.set here?

Reply:
https://vuejs.org/v2/api/#Vue-set

"ensuring the new property is also reactive, so triggers view updates."

Reply:
Sorry, what I meant was why was it necessary here when it is not necessary for the others. I assume its because it is an array now.